src/test/store/receive-profile.test.js

```diff
@@ -1069,7 +1050,7 @@ describe('actions/receive-profile', function () {
 
     it('fails in case the fetch returns a server error', async function () {
       const hash = 'c5e53f9ab6aecef926d4be68c84f2de550e2ac2f';
-      window.fetch = jest.fn().mockResolvedValue(fetch500Response);
+      window.fetch.any(500);
```

Comment:
It took me a while why we needed to use `any` here (because we have another `.catch` call in `beforeEach` block. I was going to recommend using catch but then realized this one. Maybe it could be good to add a comment that explains this here?

Reply:
The fundamental difference between `catch` and `any` is that when `catch` is reached a warning will be emitted (which is configurable). Ideally I'd like to avoid that, so ideally it would be good to have routes for every request that's supposed to happen in a test. I think that in the future I'd rather add a `.catch({ throws: new Error('No route has been defined for this request') })` in setup.js...

`any` is different: this is an expected request (even if it catches everything). I guess I was a bit lazy at the end of the day, and could have used a `get` with the full expected hash... I'm changing this :-)